concourse/model/traits/scan_sources.py

```diff
@@ -174,6 +192,13 @@ def exclude_path_regexes(self):
         default=(),
         doc='if present, perform whitesource scanning',
     ),
+    # TODO required
+    AttributeSpec.optional(
```

Comment:
actually, I would prefer to mv the compliance-db-cfg one level up - this way, it only needs to be configured once, and will be honoured by all configured scans

Reply:
Thought about the same, will do.